locales/fr.yaml

```diff
@@ -198,10 +198,10 @@ dispute_started_channel: |
 
   @${initiatorUser.username} a été impliqué.e dans ${initiatorUser.disputes} litiges
   @${counterPartyUser.username} a été impliqué.e dans ${counterPartyUser.disputes} litiges
-you_started_dispute_to_buyer: 🥴 Tu as initié un litige à propos de ton offre d'achat, nous allons te contacter toi et ta contrepartie en vue de l'arbitrage.
-buyer_started_dispute_to_seller: "🥴 L'acheteur a initié un litige concernant l'offre ID: ${orderId}, nous allons te contacter toi et ta contrepartie en vue de résoudre le problème."
-you_started_dispute_to_seller: 🥴 Tu as initié un litige à propos de ton offre de vente, nous allons te contacter toi et ta contrepartie en vue de résoudre le problème.
-seller_started_dispute_to_buyer: '🥴 Le vendeur a initié un ilitige concernant ton offre ID: ${orderId}, nous allons te contacter toi et ta contrepartie en vue de résoudre le problème.'
+you_started_dispute_to_buyer: 🥴 Tu as initié un litige à propos de ton offre d'achat. Attendez que le bot vous communique le nom d'utilisateur du Solver assigné, seul Lui/Elle pourra vous aider. Écrivez-Lui/Elle. S'il/Elle vous contacte en premier, vérifiez que son nom d'utilisateur correspond à celui que le bot vous a donné.
+buyer_started_dispute_to_seller: '🥴 L'acheteur a initié un litige concernant l'offre ID: ${orderId}. Attendez que le bot vous communique le nom d'utilisateur du Solver assigné, seul Lui/Elle pourra vous aider. Écrivez-Lui/Elle. S'il/Elle vous contacte en premier, vérifiez que son nom d'utilisateur correspond à celui que le bot vous a donné.'
```

Comment:
@Catrya beware, if you change double-quotes to single-quotes, you break parsing (this is why CI is breaking now), because the text already has single-quotes inside, so you cannot use this char to wrap the text

Reply:
Thank you very much, I hadn't noticed, I'll fix it.

Reply:
Thank you very much, I hadn't noticed, I'll fix it.